torch/csrc/dynamo/cpython_defs.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+// Functions that need to be copied from the CPython source
+// should go in cpython_defs.c. Copying is required when, e.g.,
+// we need to call internal CPython functions that are not exposed.
+
+#if IS_PYTHON_3_11_PLUS
+
+#include <internal/pycore_frame.h>
+
+static int
+_PyFrame_OpAlreadyRan(_PyInterpreterFrame *frame, int opcode, int oparg);
+
+int
+THP_PyFrame_FastToLocalsWithError(_PyInterpreterFrame *frame);
```

Comment:
Wouldn't hurt to link to copied-from source, ex: https://github.com/python/cpython/blob/main/Objects/frameobject.c#L1103

Reply:
There is a detailed comment and link in the .c file
Do you think the header is a better place to put it?